features/issue.feature

```diff
@@ -80,6 +80,34 @@ Feature: hub issue
     """
     When I successfully run `hub issue -d 2016-08-18T09:11:32Z`
 
+  Scenario: Fetch issues
```

Comment:
The scenario name needs to be more specific, like "Fetch issues sorted by number of comments ascending"